lib/zip/extra_field.rb

```diff
@@ -69,7 +69,7 @@ def to_local_bin
       ordered_values.map! { |v| v.to_local_bin.force_encoding('BINARY') }.join
     end
 
-    alias :to_s :to_local_bin
+    alias_method :to_s, :to_local_bin
```

Comment:
There is no sense to use alias_method because we don't need chains. So better use alias instead of alias_method